eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/EventMeshServer.java

```diff
@@ -134,6 +150,29 @@ public void init() throws Exception {
             }
         }
 
+        if (Objects.nonNull(eventMeshTCPServer)) {
+            MetricsManager metricsManager = eventMeshTCPServer.getEventMeshTcpMetricsManager();
+            this.eventMeshMetricsManager.addMetricManager(metricsManager);
+            this.eventMeshMetricsManager.addMetrics(metricsManager.getMetrics());
+        }
+
+        if (Objects.nonNull(eventMeshGrpcServer)) {
+            MetricsManager metricsManager = eventMeshGrpcServer.getEventMeshGrpcMetricsManager();
+            this.eventMeshMetricsManager.addMetricManager(metricsManager);
+            this.eventMeshMetricsManager.addMetrics(metricsManager.getMetrics());
+        }
+
+        if (Objects.nonNull(eventMeshHTTPServer)) {
+            MetricsManager metricsManager = eventMeshHTTPServer.getEventMeshHttpMetricsManager();
+            this.eventMeshMetricsManager.addMetricManager(metricsManager);
+            this.eventMeshMetricsManager.addMetrics(metricsManager.getMetrics());
+        }
```

Comment:
The same `addMetricManager` and `addMetrics` methods are repeated three times, and can be extracted outside of the if condition is not that strict; of course, if the if condition is very strict, it can be left as is.

Reply:
> The same `addMetricManager` and `addMetrics` methods are repeated three times, and can be extracted outside of the if condition is not that strict; of course, if the if condition is very strict, it can be left as is.

oh! I will fix it

Reply:
Fixed in https://github.com/apache/eventmesh/pull/4840/commits/1508e4c24c9d2d78b696b50e93f5bbf3b7dd1446